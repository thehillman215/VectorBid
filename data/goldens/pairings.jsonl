{"pairing_id":"EWR-73N-001","base":"EWR","fleet":"73N","month":"2025-09-01","trips":[{"trip_id":"EWR-73N-001-T1","pairing_id":"EWR-73N-001","day":5,"origin":"DEN","destination":"EWR"},{"trip_id":"EWR-73N-001-T2","pairing_id":"EWR-73N-001","day":3,"origin":"LAX","destination":"EWR"}]}
