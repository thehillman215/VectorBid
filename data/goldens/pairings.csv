pairing_id,base,fleet,month
EWR-73N-001,EWR,73N,2025-09-01
